accounts/abi/reflect.go

```diff
@@ -38,11 +38,13 @@ import (
 // given type
 // e.g. turn
 // var fields []reflect.StructField
-// fields = append(fields, reflect.StructField{
-// 		Name: "X",
-//		Type: reflect.TypeOf(new(big.Int)),
-//		Tag:  reflect.StructTag("json:\"" + "x" + "\""),
-// }
+//
```

Comment:
can we add a tab to the above line and remove this additional newline?

Reply:
done thank you